### ISSUE TITLE
Cloning should comply w Tiddler opening behaviour

### ISSUE THREAD

**Issue by twMat** [Jermolene/TiddlyWiki5#3455]
When setting "Tiddler opening behaviour" is set to (default):

Tiddler Opening Behaviour > Navigation from within the story river : Open below the current tiddler

...and using tiddler tool "Clone tiddler" this opens clone at top of river.

**Proposal**
Make cloning navigation behaviour comply with set Tiddler Opening Behaviour

**Comment by BurningTreeC** [comment]
@twMat yes I think it's misleading because openLinkFromOutsideRiver makes cloned tiddlers show at the bottom
